src/main/java/com/microsoft/sqlserver/jdbc/SQLServerSavepoint.java

```diff
@@ -9,7 +9,7 @@
 
 
 /**
- * SQLServerSavepoint implements JDBC 3.0 savepoints. A savepoint is checkpoint to which a transaction can be rolled
+ * Provides an implementation of the JDBC 3.0 savepoints. A savepoint is checkpoint to which a transaction can be rolled
```

Comment:
`JDBC 3.0 savepoints` can be replaced by _JDBC Interface <code>java.sql.Savepoint</code>_